parlai/crowdsourcing/utils/analysis.py

```diff
@@ -79,3 +82,65 @@ def __init__(self, opt: Dict[str, Any]):
             raise ValueError(
                 'There must be a "none_all_good" category in self.problem_buckets!'
             )
+
+    @abstractmethod
+    def compile_results(self) -> pd.DataFrame:
```

Comment:
I missed this method by mistake while merging. I will remove this as it is already in the parent class.